erpnext/selling/doctype/quoting_sheet/quoting_sheet.js

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2021, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+frappe.ui.form.on('Quoting Sheet', {
+	// refresh: function(frm) {
+	// },
+	item_code: function (frm) {
+		if (frm.doc.currency == frappe.sys_defaults.currency) {
+			frm.set_value("conversion_rate", 1.0)
+		}
+	},
+
+	invoke_doc_function(frm, method) {
+		frappe.call({
+			doc: frm.doc,
+			method: method,
+			callback: function (r) {
+				if (!r.exe) {
+					frm.refresh_fields();
+				}
+			}
+		});
+	},
```

Comment:
- Get rid of this function, this is unnecessary.
- Merge this with `calculate`.